Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -183,7 +184,8 @@ class BrowserViewController: UIViewController {
             home.view.setNeedsUpdateConstraints()
         }
 
-        if let tab = tabManager.selectedTab,
+        if !isInBackground,
```

Comment:
Yeah, I was just saying that if the specific path you hit is problematic, there are a number of other ways we can hit this same code while in the background. For example, we'll call `updateUIForReaderHomeStateForTab` (which in turn calls `updateURLBarDisplayURL`) [any time the URL changes](https://github.com/mozilla/firefox-ios/blob/4fc8cade687a8eb07e2f5c914616872dc0d13baf/Client/Frontend/Browser/BrowserViewController.swift#L889), which can happen with redirects or delayed events. 

I'd lean toward "not caring about this commit" unless we're careful to avoid touching this code everywhere we're in the background (which would probably be difficult, as I imagine there are many other places outside BVC that could trigger similar queries).